adoc/admin-troubleshooting.adoc

```diff
@@ -0,0 +1,46 @@
+= Troubleshooting
+
+This chapter summarizes frequent problems that can occur while using {productname}
+and their solutions.
+
+Additionally, {suse} support collects problems and their solutions online at link:https://www.suse.com/support/kb/?id=SUSE_CaaS_Platform[] .
+
+== The `supportconfig` Tool
+
+As a first step for any troubleshooting/debugging effort you need to find out
+where the problem is caused. For this purpose we ship the `supportconfig` tool
+and plugin with {productname}. With a simple command you can collect and compile
+a variety of details about your cluster to enable {suse} support to pinpoint
+the potential cause of an issue.
+
+In case of problems, a detailed system report can be created with the `supportconfig` command line tool.
+It will collect information about the system such as:
+
+* Current Kernel version
+* Hardware information
+* Installed packages
+* Partition setup
+* Cluster and node status
+
+[TIP]
+====
+A full list of of the data collected by `supportconfig` can be found link:https://github.com/SUSE/supportutils-plugin-suse-caasp/blob/master/README.md[here].
+====
+
+To collect all relevant logs run the `supportconfig` command on all the master and worker nodes individually.
+
+[source,bash]
+----
+supportconfig
+tar -xvjf /var/log/nts_*.tbz
```

Comment:
Errata: in my case `supportconfig` produced "txz" archive filetype - so it's not tar.bz2 archive but tar.xz archive. For working with xz archives tar uses `-J` (capital J)

```suggestion
tar -xvJf /var/log/nts_*.txz
```

Reply:
Thanks, fixed it here: https://github.com/SUSE/doc-caasp/pull/350